index.js

```diff
@@ -4,7 +4,32 @@
  * Depending on the operation, either add up all of the numbers or subtract all of the numbers, from left to right.
  * @returns {number} The result of either adding all numbers or subtracting all numbers, depending on the arguments added to the command line.
  */
-function calculator() {}
+function calculator() {   
+    let arr = process.argv.slice(2);
+    let sum = Number(arr[1]);
```

Comment:
Since you can subtract and add a more accurate variable name would be something like `total` or `result`